### ISSUE TITLE
Set album photo sort order with new API

### ISSUE THREAD

**Issue by EddyLB** [Piwigo/Piwigo#918]
Following #917 , could it be also possible to create an API with which an external app could change the album photo sort order ?

**Comment by plegall** [comment]
Yes, absolutely. As I see Piwigo 3.0, all admin features will be available in the API (making them available to 3rd party tools like the iOS app)
